demo/src/index.js

```diff
@@ -1,48 +1,90 @@
-import React from 'react'
-import {render} from 'react-dom'
-import ReactHintFactory from 'src'
-import 'css/index.css'
-import './index.css'
+import React from "react";
```

Comment:
Could you use the same code styling? E.g. single quotes instead of double quotes, no semicolons, etc?

Reply:
Can you add prettier config file with your preferred style?